### ISSUE TITLE
periorificial hyperkeratosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9609]
**Preferred term label:**
Periorificial hyperkeratosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Scaling and overgrowth of horny tissue of the skin surrounding body orifices (including peri-auricular, peri-oral, peri-anal areas.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614594 
PMID: 22405088 6/6

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
